### PR TITLE
adding data-* to all of the tests

### PR DESCRIPTION
**Pull request by SaharKargar**
Resolved issues: 
https://github.com/project-chip/zap/issues/639
https://github.com/project-chip/zap/issues/538

### CODE REVIEW

**Review by pedramk — APPROVED** [review]

File: src/components/ZclEndpointCard.vue [hunk 1]
```diff
@@ -316,7 +294,7 @@ export default {
               }
             }
           })
-          this.selectedservers = [...enabledServers, ...enabledServers]
+          this.selectedservers = [...enabledServers, ...enabledClients]
```
Comment (jingteng25742) [inline src/components/ZclEndpointCard.vue:297]: mistake?

File: Jenkinsfile [hunk 2]
```diff
@@ -47,6 +47,8 @@ pipeline
                         {
                                     sh 'npm run version-stamp'
                                     sh 'npm run version'
+                                    sh 'npm rebuild canvas --update-binary'
```
Comment (jingteng25742) [inline Jenkinsfile:50]: @SaharKargar 
Can you rebase your changes? 
There are a few changes that is not related to data-* work.

File: apack.json [hunk 3]
```diff
@@ -19,6 +19,10 @@
       "exe": "zap.app/Contents/MacOS/zap",
       "optional": true
     },
+    "zap:macosx.ARM64": {
```
Comment (jingteng25742) [inline apack.json:22]: @SaharKargar
Can you rebase your changes?
There are a few changes that is not related to data-* work.

File: src-electron/db/query-package.js [hunk 4]
```diff
@@ -896,6 +915,7 @@ exports.getPackageSessionPackagePairBySessionId =
   getPackageSessionPackagePairBySessionId
 
 exports.getPathCrc = getPathCrc
+exports.getZclPropertiesPackage = getZclPropertiesPackage
```
Comment (jingteng25742) [inline src-electron/db/query-package.js:918]: @SaharKargar
Can you rebase your changes?
There are a few changes that is not related to data-* work.

File: src-script/download-artifact.ts [hunk 5]
```diff
@@ -20,7 +20,7 @@
 // This util script downloads ZAP artifact from Github.com
 // Usage: node ./download-artifact.js $branch [ $commit | latest ]
 
-import { Octokit, App } from 'octokit'
+import { Octokit } from 'octokit'
```
Comment (jingteng25742) [inline src-script/download-artifact.ts:23]: @SaharKargar
Can you rebase your changes?
There are a few changes that is not related to data-* work.

**Review by jingteng25742 — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 2 above)
Comment (SaharKargar) [reply Jenkinsfile:50]: @jingteng25742 
Rebased and pushed.

File: apack.json (hunk 3 above)
Comment (SaharKargar) [reply apack.json:22]: @jingteng25742 
Rebased and pushed.

File: src-electron/db/query-package.js (hunk 4 above)
Comment (SaharKargar) [reply src-electron/db/query-package.js:918]: @jingteng25742 
Rebased and pushed.

File: src-script/download-artifact.ts (hunk 5 above)
Comment (SaharKargar) [reply src-script/download-artifact.ts:23]: @jingteng25742 
Rebased and pushed.

File: src/components/ZclEndpointCard.vue (hunk 1 above)
Comment (Amirreza-Ghasemkhani) [reply src/components/ZclEndpointCard.vue:297]: It was typo and its fixed right now
